### ISSUE TITLE
R installation points to v0.1.7 when requestion v0.1.9

### ISSUE THREAD

**Issue by cboettig** [cwida/duckdb#769]
Run the [documented](https://duckdb.org/) installation method for R:

```r
install.packages("duckdb", repos=c("http://download.duckdb.org/alias/v0.1.9/rstats/", "http://cran.rstudio.com"))
```
repo suggests this is v0.1.9, but installer reports:

```
trying URL 'http://download.duckdb.org/alias/v0.1.9/rstats/src/contrib/duckdb_0.1.7.tar.gz'
```

Same issue can be seen in browsing from http://download.duckdb.org/, e.g. 0.1.9 link takes you to:

http://download.duckdb.org/rev/3c353fac49d37719d7592d1819dfeb76ac541736/rstats/

Looks like this may have been flagged with other issues in #711.  I don't experience the build/install issue mentioned there though, only the version issue

**Comment by hannesmuehleisen** [comment]
Yes, there is some version number confusion here. We are working on getting `duckdb` on CRAN, so that should be cleared up in that effort as well.

**Comment by hannesmuehleisen** [comment]
This should now be fixed for now with the release of 0.2.0
